Formula/infer.rb

```diff
@@ -127,8 +95,30 @@ def install
       }
     EOS
 
-    shell_output("#{bin}/infer --fail-on-issue -P -- clang -c FailingTest.c", 2)
-    shell_output("#{bin}/infer --fail-on-issue -P -- clang -c PassingTest.c")
+    no_issues_output = "\n  No issues found  \n"
+
+    failing_c_output = <<~EOS
+
+FailingTest.c:5: error: Null Dereference
+  pointer `s` last assigned on line 4 could be null and is dereferenced at line 5, column 3.
+  3. int main() {
+  4.   int *s = NULL;
+  5.   *s = 42;
+       ^
+  6. 
```

Comment:
`brew audit` is probably going to complain about trailing whitespace here. You could just get rid of the whitespace in `FailingTest.c` to avoid the need for this line.

Reply:
Ah! Good idea! That fixed that one!